_projects/permaculture-design-tools.md

```diff
@@ -7,6 +7,6 @@ labels:
 - Business Tools
 - Open Data
 current_stage: Project Pitch
```

Comment:
Totally! That is part of some _Writings_ I'd like to link at some point with projects. It's part of uncommitted changes which are on my computer at the moment – so another PR will likely address this question :-) I see a lot of value in making the pitch public